lib/appsignal/config.rb

```diff
@@ -90,13 +91,18 @@ def []=(key, value)
 
     def log_file_path
       path = config_hash[:log_path] || root_path
-      if path && File.writable?(path)
-        File.join(File.realpath(path), 'appsignal.log')
+      return File.join(path, 'appsignal.log') if path && File.writable?(path)
+
+      if File.writable? SYSTEM_TMP_DIR
+        $stdout.puts "appsignal: Unable to log to '#{path}'. Logging to "\
+          "'#{SYSTEM_TMP_DIR}' instead. Please check the "\
+          "permissions for the application's log directory."
+        File.join(SYSTEM_TMP_DIR, 'appsignal.log')
       else
-        '/tmp/appsignal.log'
+        $stdout.puts "appsignal: Unable to log to '#{path}' or the "\
+          "'#{SYSTEM_TMP_DIR}' fallback. Please check the permissions "\
+          "for the application's (log) directory."
       end
-    rescue Errno::ENOENT
```

Comment:
I removed it because I'm not quite sure what can cause an Errno::ENOENT to be thrown here.

File.writable? doesn't throw it for a file that doesn't exist or is not writable.